develop/tutorials/articles/160-search/03-indexers.markdown

```diff
@@ -67,7 +67,13 @@ For example, the
 [`GroupedModelDocumentContributor`](https://github.com/liferay/liferay-portal/blob/7.1.1-ga2/modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/contributor/document/GroupedModelDocumentContributor.java)
 contributes the `GROUP_ID` and `SCOPE_GROUP_ID` fields for all documents with a
 backing entity that's also a `GroupedModel`.
- 
+
```

Comment:
Doesn't this duplicate what's directly above? :point_up:

Reply:
See https://dev.liferay.com/en/develop/tutorials/-/knowledge_base/7-1/indexing-framework#indexing, the last paragraph.

Reply:
Yeah, though it's better to explain what "type" means. It sounds vague to me.

Reply:
Well I was actually referring to the type parameter in the class declaration, so I'll add the part about the component property, too